complex_spatial_process.ipynb

```diff
@@ -0,0 +1,1071 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #35.                if single_burn &gt;= 0.005:
</pre><p>can you add a comment about 0.005 and why you chose that value?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/WAT-ai/wildfire/pull/34/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='9'/>